source/base/parameter_handler.cc

```diff
@@ -1382,10 +1382,44 @@ bool ParameterHandler::read_input (const std::string &filename,
   try
     {
       std::string openname = search.find(filename);
-      std::ifstream input (openname.c_str());
-      AssertThrow(input, ExcIO());
+      std::ifstream file (openname.c_str());
+      AssertThrow(file, ExcIO());
+
+      // concatenate the lines ending with "\"
+      std::string str;
+      std::string file_contents;
+      bool is_concatenated(false);
+      while (std::getline(file, str))
+        {
+          // if this str must be concatenated with the previous one
+          // we strip the whitespaces
+          if (is_concatenated)
+            {
+              const std::size_t str_begin = str.find_first_not_of(" \t");
+              if (str_begin == std::string::npos)
+                return ""; // no content
```

Comment:
I don't think you want to exit the entire function just because you've found an empty continuation line :-)